gum/backend-arm64/gumstalker-arm64-glue.S

```diff
@@ -0,0 +1,13 @@
+    .text
+    .align  2
+
+    .global gum_stalker_follow_me
+    .type   gum_stalker_follow_me, %function
+gum_stalker_follow_me:
+    stp	x29, x30, [sp,-16]!
+    mov	x29, sp
+    mov x2, x30
+    bl _gum_stalker_do_follow_me
+    ldp	x29, x30, [sp],16
```

Comment:
Extra spaces here, let's not align the assembly code but rather go for a stable layout.